ta/pkcs11/src/processing_symm.c

```diff
@@ -339,6 +412,33 @@ enum pkcs11_rc step_symm_operation(struct pkcs11_session *session,
 	 * (PKCS11_FUNC_STEP_UPDATE/_ONESHOT)
```

Comment:
Need to update comment : inconsistent with lines 421/422 :(
I suggest to remove line 339 and only keep line 338 ` * Feed active operation with data`.

Reply:
Done